pkg/context/helpers.go

```diff
@@ -133,6 +145,14 @@ func (c *TestContext) AddTestFileContents(file []byte, dest string) error {
 	return c.FileSystem.WriteFile(dest, file, os.ModePerm)
 }
 
+// AddTestDirectoryFromRoot adds a test directory where the filepath is relative to the root of the repository.
+// Use this when the directory you are referencing is in a different directory than the test.
+func (c *TestContext) AddTestDirectoryFromRoot(srcDir, destDir string) {
+	pathFromRoot := filepath.Join(c.FindRepoRoot(), srcDir)
+	c.AddTestDirectory(pathFromRoot, destDir)
+}
+
+// AddTestDirectory adds a test directory where the filepath is relative to the test directory.
```

Comment:
Do we want a similar check if srcDir contains `..` and call `T.Fatal(..)`?

Reply:
Oh yeah! 👍 The goal of these is to let us use a path to the test file that doesn't get messed up as much when the test is moved. Relative paths are still vulnerable but usually the testdata dir moves with a test.